src/validators/output/index.test.js

```diff
@@ -0,0 +1,68 @@
+import test from 'ava'
+import outputValidator from '.'
+
+const absolutePath = process.platform === 'win32' ? 'C://foo/bar/baz' : '/foo/bar/baz'
+const relativePath = 'foo/bar'
+
+
+outputValidator.forEach((item) => {
```

Comment:
Maybe we could make this simpler by exporting each validator above like:

```javascript
// output/index.js
export function validateOutputPath(value) {
  //...
}

// output/index.test.js
import * as validators from '.'

runOutputPathTests(validators.validateOutputPath)
```

Thoughts?